lua/lazy/view/render.lua

```diff
@@ -499,7 +499,11 @@ function M:details(plugin)
     if git.branch then
       table.insert(props, { "branch", git.branch })
     end
-    table.insert(props, { "commit", git.commit:sub(1, 7), "LazyCommit" })
+    if git.commit then
+      table.insert(props, { "commit", git.commit:sub(1, 7), "LazyCommit" })
+    else
+      table.insert(props, { "commit", "No commits yet" })
```

Comment:
Maybe better to just not show the commit here in this case? So just drop the `else` part?

Reply:
In my case, `git.commit` was `nil` but `git.branch` was not `nil`, so I thought it would be better to show the commit as well, just in case.
However, it is not always the case that `git.commit` does not become `nil` in other cases, so I should certainly drop the `else` part.